spec/spec_helper.rb

```diff
@@ -17,6 +17,7 @@ class GrpcApp < Rails::Application
 loader.ignore("#{Rails.root}/spec/test_service_pb.rb")
 loader.setup
 require "#{Rails.root}/spec/test_service_pb.rb"
+require "#{Rails.root}/lib/base_interceptor.rb"
```

Comment:
Not sure if this was the right thing to do, but I had to include this so my tests could properly run on the CI.